### PR TITLE
Css fixes

### PR DESCRIPTION
**Pull request by braedons**
**Summary**
Just some simple css fixes:
- Left and right toolbars are now the same fixed height
- Settings pop up is widened (to a fixed width) to fit the content fully
  - Note: It's a little wide depending on the resolution. I'm not sure if the way I did it is the best
- Added spacing between buttons in: 
  - Calendar pane toolbar
  - SectionTable course info
  - Copy and Clear Schedule in Added Classes tab

image
image
image
image

**Test Plan**
- Check that the button spacing in each area is adequate
- Try different zooms to see how it'd behave on different screen sizes (for the toolbar height and the settings pop up width)

**Issues**
#36

### CODE REVIEW

File: client/src/components/AddedCourses/AddedCoursePane.js [hunk 1]
```diff
@@ -20,6 +20,10 @@ const styles = {
         width: '100%',
         justifyContent: 'space-between',
     },
+    clearSchedule: {
+        marginLeft: '4px',
+        marginRight: '4px',
```
Comment (ChaseC99) [inline client/src/components/AddedCourses/AddedCoursePane.js:25]: This could also be condensed down to: `margin: auto 4px` 

There are 4 main ways you can add margin
```
margin: 4px;                       // 4px to top, right, bottom, and center
margin: 0 4px;                    // 0px top/bottom, 4px left/right
margin: 1px 2px 3px 4px;         // 1px top, 2px right, 3px bottom, 4px left
margin-left: 4px;               // Specify which side you wanna add the margin too.
```

Don't necessarily have to change it. You might even argue that marginLeft/marginRight is more readable. I just wanted to point it out for learning.

File: client/src/components/Calendar/CalendarPaneToolbar.js [hunk 2]
```diff
@@ -14,6 +14,11 @@ const styles = {
         overflow: 'auto',
         marginBottom: '4px',
         alignItems: 'center',
+        height: '48px',
+
+        '& button': {
```
Comment (ChaseC99) [inline client/src/components/Calendar/CalendarPaneToolbar.js:19]: I'm not familiar with this syntax. What does the `&` do?

File: package.json [hunk 3]
```diff
@@ -28,6 +28,7 @@
         ]
     },
     "dependencies": {
-        "notistack": "^1.0.2"
+        "notistack": "^1.0.2",
+        "react-lazyload": "^3.1.0"
```
Comment (ChaseC99) [inline package.json:32]: Why was this added to our package?

File: client/src/components/App/SettingsMenu.js [hunk 4]
```diff
@@ -4,11 +4,12 @@ import { Settings } from '@material-ui/icons';
 import AppStore from '../../stores/AppStore';
 import { toggleDarkMode } from '../../actions/AppStoreActions';
 import { withStyles } from '@material-ui/core/styles';
+import classNames from 'classnames';
```
Comment (ChaseC99) [inline client/src/components/App/SettingsMenu.js:7]: I don't think we use `classnames` in this file. We can go ahead and remove that.

**Review by ChaseC99 — APPROVED** [review]
Site is looking really good 😍 🔥 

I left a few comments but this lgtm! Great work 👍

File: package.json (hunk 3 above)
Comment (ChaseC99) [reply package.json:32]: We use it in client, but we shouldn't need that dependency out here in the root.

**Comment by ChaseC99** [comment]
Oh looks like there's a merge conflict with some of @devsdevsdevs code. Might want to sync with him to resolve that properly.

File: client/src/components/Calendar/CalendarPaneToolbar.js (hunk 2 above)
Comment (braedons) [reply client/src/components/Calendar/CalendarPaneToolbar.js:19]: It lets you do nesting. I don't think it's vanilla css.

File: package.json (hunk 3 above)
Comment (braedons) [reply package.json:32]: I was having trouble starting the server until I installed it. It was throwing an error about it missing. But I suppose I should have added it to client/package.json?

File: client/src/components/App/SettingsMenu.js (hunk 4 above)
Comment (braedons) [reply client/src/components/App/SettingsMenu.js:7]: Good catch. I guess I thought I was going to use it, but ended up not. Will remove.

File: package.json (hunk 3 above)
Comment (braedons) [reply package.json:32]: Nevermind, tried removing it and it was fine. I'll keep it out.
